concepticondata/conceptlists/Petroni-2010-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	S_I
+Petroni-2010-100-1	1	You	1215	THOU	0.45395
+Petroni-2010-100-2	2	Three	492	THREE	0.44102
+Petroni-2010-100-3	3	Mother	1216	MOTHER	0.36627
+Petroni-2010-100-4	4	Not	1240	NOT	0.35033
+Petroni-2010-100-5	5	New	1231	NEW	0.31961
+Petroni-2010-100-6	6	Nose	1221	NOSE	0.3169
+Petroni-2010-100-7	7	Four	1500	FOUR	0.30226
+Petroni-2010-100-8	8	Night	1233	NIGHT	0.29403
+Petroni-2010-100-9	9	Two	1498	TWO	0.28214
+Petroni-2010-100-10	10	Name	1405	NAME	0.27962
+Petroni-2010-100-11	11	Tooth	1380	TOOTH	0.27677
+Petroni-2010-100-12	12	Star	1430	STAR	0.27269
+Petroni-2010-100-13	13	Salt	1274	SALT	0.26792
+Petroni-2010-100-14	14	Day	1225	DAY (NOT NIGHT)	0.26695
+Petroni-2010-100-15	15	Grass	606	GRASS	0.26231
+Petroni-2010-100-16	16	Sea	1474	SEA	0.25906
+Petroni-2010-100-17	17	Die	1494	DIE	0.25602
+Petroni-2010-100-18	18	Sun	1343	SUN	0.25535
+Petroni-2010-100-19	19	One	1493	ONE	0.23093
+Petroni-2010-100-20	20	Feather	1201	FEATHER	0.23055
+Petroni-2010-100-21	21	Give	1447	GIVE	0.22864
+Petroni-2010-100-22	22	Sit	1416	SIT	0.22757
+Petroni-2010-100-23	23	Stand	1442	STAND	0.22644
+Petroni-2010-100-24	24	Meat	634	MEAT	0.2261
+Petroni-2010-100-25	25	Long	1203	LONG	0.22491
+Petroni-2010-100-26	26	Five	493	FIVE	0.22353
+Petroni-2010-100-27	27	Hand	1277	HAND	0.22261
+Petroni-2010-100-28	28	Short	1645	SHORT	0.21676
+Petroni-2010-100-29	29	Father	1217	FATHER	0.21319
+Petroni-2010-100-30	30	Smoke	778	SMOKE (EXHAUST)	0.21213
+Petroni-2010-100-31	31	Far	1406	FAR	0.20998
+Petroni-2010-100-32	32	Worm	1219	WORM	0.20846
+Petroni-2010-100-33	33	Dry	1398	DRY	0.207
+Petroni-2010-100-34	34	Scratch	1436	SCRATCH	0.20343
+Petroni-2010-100-35	35	Person	683	PERSON	0.20129
+Petroni-2010-100-36	36	When	1238	WHEN	0.20011
+Petroni-2010-100-37	37	Wind	960	WIND	0.19535
+Petroni-2010-100-38	38	Snake	730	SNAKE	0.19485
+Petroni-2010-100-39	39	Sing	1261	SING	0.19434
+Petroni-2010-100-40	40	Stone	857	STONE	0.19369
+Petroni-2010-100-41	41	Suck	1421	SUCK	0.19196
+Petroni-2010-100-42	42	Mouth	674	MOUTH	0.19067
+Petroni-2010-100-43	43	Dig	1418	DIG	0.19052
+Petroni-2010-100-44	44	Live	1422	BE ALIVE	0.18716
+Petroni-2010-100-45	45	Root	670	ROOT	0.18715
+Petroni-2010-100-46	46	Hair	2648	HAIR (HEAD)	0.18522
```

Comment:
Is this specific, i.e. HEAD HAIR, or just HAIR?

Reply:
yes, if they claim, it's the swadesh-200, it shoudl be the one they have there.

Reply:
@SimonGreenhill @LinguList @chrzyki thank you for your help!